index.d.ts

```diff
@@ -1668,6 +1672,54 @@ declare namespace Moleculer {
 		}
 	}
 
+	/**
+	 * you can use multiple modifier with a dot
+	 * e.g. `black.bgRed.bold`
+	 */
+	type KleurColor = keyof Kleur | string;
```

Comment:
Is union with `string` necessary here, or should the colors all be from `kleur`?

Reply:
KleurColor can be linked together . like I said on the comment . 

So : `${Kleur.black}` is valid, `${Kleur.black}.${Kleur.bgRed}` too, etc

Reply:
I wouldn't think it worth it to go down the path of making a recursive type.

Unfortunately, once you union the `keyof Kleur` with `string` its going to widen to `string`, effectively rendering the `keyof Kleur` moot.  If allowing `string` is important, you can likely just allow `string` without the `keyof Kleur` because it ends up in the same place anyway.

Reply:
the union, allow intellisense without limiting too : 
![image](https://user-images.githubusercontent.com/5489218/226192463-b0524dab-e4a6-42ee-b323-509dd47ad2f6.png)